pom.xml

```diff
@@ -16,8 +16,8 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <jackson.version>2.5.4</jackson.version>
-    <azuresdk.version>0.9.4</azuresdk.version>
+    <azuresdk.version>1.0.0-beta4.1</azuresdk.version>
+    <azure-storage.version>4.4.0</azure-storage.version>
     <jenkins.version>1.653</jenkins.version>
```

Comment:
Why do we think we need that.? We're still compatible with the current minimum version.